aries-site/src/pages/foundation/background-colors-guidance.mdx

```diff
@@ -65,14 +93,31 @@ above a main content section that uses `background-back`.
 Refer to the [background palette](/foundation/color#background-palette) for definition
 of each background color namespace and value.
 
-### Example of layered layout
+## Examples of Layered Layout
 
 For example, a page with a [header](/components/header), summary content, and a grid of
 [cards](/components/card) would use `background-front` for each card and
 `background-back` for everything else.
 
 For a multi-column page layout, `background-front` could be used to distinguish the column which should be the primary focus.
 
+### With fixed header
+
+When the header is fixed, use `background-front` for the header since it is layered in front of the main content.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/foundation/background-colors/LayeredLayoutFixedExample.js"
+  width="100%"
+  showResponsiveControls={['laptop', 'mobile']}
+  screenContainer
+>
+  <LayeredLayoutFixedExample />
+</Example>
+
+### Header scrolls with content
+
+When the header scrolls with the main content, use `background-back` for the header since it is flush with the main content. An example of this design would be the Design System website.
```

Comment:
Same question as above.

```suggestion
When the header scrolls with the main content, use `background-back` <ColorSwatch background="background-back" /> for the header since it is flush with the main content. An example of this design would be the Design System website.
```